netutils/ntpclient/ntpclient.c

```diff
@@ -577,6 +1503,23 @@ int ntpc_start(void)
   return g_ntpc_daemon.pid;
 }
 
+/****************************************************************************
+ * Name: ntpc_start
+ *
+ * Description:
+ *   Start the NTP daemon
+ *
+ * Returned Value:
+ *   On success, the non-negative task ID of the NTPC daemon is returned;
+ *   On failure, a negated errno value is returned.
+ *
+ ****************************************************************************/
+
+int ntpc_start(void)
+{
+  return ntpc_start_with_list(NULL);
```

Comment:
should we change NULL to CONFIG_NETUTILS_NTPCLIENT_SERVER? and remvoe the check from ntp task.

Reply:
Sounds like a good idea.